html-aam/index.html

```diff
@@ -1257,6 +1257,79 @@ <h4 id="el-button">`button`</h4>
+                     its purpose is directly tied to the rendering of a customized select
+                     serving as the visible indicator/styleable node to serve as the 
+                     invoking element for the select's listbox popup.
+                -->
+              </td>
+            </tr>
+            <tr>
+              <th><a data-cite="core-aam-1.2/#roleMappingComputedRole">Computed Role</a></th>
+              <td class="role-computed">
+                <div class="general"><a href="#el-button-select-child-comments">See comments</a>.</div>
+              </td>
+            </tr>
+            <tr>
+              <th>
+                <a href="https://msdn.microsoft.com/en-us/library/dd373608%28v=VS.85%29.aspx">MSAA</a> + <a href="http://accessibility.linuxfoundation.org/a11yspecs/ia2/docs/html/">IAccessible2</a>
+              </th>
+              <td>
+sdfsddsaf
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://msdn.microsoft.com/en-us/library/ms726297%28v=VS.85%29.aspx">UIA</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://gnome.pages.gitlab.gnome.org/atk/">ATK</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <tr>
+              <th><a href="https://developer.apple.com/reference/appkit/nsaccessibility">AX</a></th>
+              <td>
+                <div class="general">Use WAI-ARIA mapping</div>
+              </td>
+            </tr>
+            <!-- <th><a href="#accessible-name-and-description-computation">Naming Algorithm</a></th> -->
+            <tr>
+              <th id="el-button-select-child-comments">Comments</th>
+              <td>
+                <p>User agents MUST treat a `button` that is a child element of a `select` element as one item  in the accessibility tree.</p>
+                <p>
+                  User agents MUST use the <a data-cite="accname-1.2/#mapping_additional_nd_te">text equivalent computation</a> of the subtree of the
+                  `button` element as the exposed value of the `select` element. If the `button` contains no content, then there is no value to expose 
+                  for the collapsed state of the `select`.
+                </p>
+                <p class=note>
+                  TBD: indicate that global html/aria attributes and html/aria attributes specific to buttons are not to be respected on a button that is a child of a select?  Since the button/select are meant to be treated as one entity in the a11y tree, and the button's content is meant to be treated as the value for the select, and not the name of the select, it doesn't make much sense to treat this button as if it were actually a button... 
```

Comment:
```suggestion
                  TBD: indicate that global html/aria attributes and html/aria attributes specific to buttons are not to be respected on a button that is a child of a select?  Since the button/select are meant to be treated as one entity in the a11y tree, and the button's content is meant to be treated as the value for the select, and not the name of the select, it doesn't make much sense to treat this button as if it were actually a button... 
            <br>
            <br>
            suggest ignoring accesskey, autofocus, contenteditable, draggable, inert, popover, tabindex, title
```